src/draw.py

```diff
@@ -0,0 +1,166 @@
+import pygame
+import random
+from heapq import *
+from graph import Graph
+
+class Plot:
+
+    screen : any
+    clock : any
+    display_size : tuple
+    graph : Graph
+    fps : int
+    nodes_data : list
+    edges_data : dict
+
+    node_radius : int
+    labels_color : tuple
+    undiscovered_node_color : tuple
+    discovered_node_outline : tuple
+    current_node_fill : tuple
+    discovered_node_fill : tuple
+    undiscovered_node_fill : tuple
+    completed_node_color : tuple
+    undiscovered_edge_color : tuple
+
+
+    def __init__(self, graph : Graph, screen, clock, fps, 
+                coords=None,
+                node_radius=30,
+                labels_color=(255, 255, 0),
+                undiscovered_node_color=(100, 100, 100), 
+                discovered_node_outline=(255, 255, 255),
+                current_node_fill=(200, 200, 0),
+                discovered_node_fill=(200,0,0),
+                undiscovered_node_fill=(0, 0, 0),
+                completed_node_color=(50,50,160),
+                undiscovered_edge_color=(100, 100, 100),
+                discovered_edge_color=(255, 255, 255)) -> None:
+        
+        self.clock = clock
+        self.screen = screen
+        self.display_size = screen.get_size()
+        self.graph = graph
+        self.fps = fps
+        self.node_radius = node_radius
+        self.nodes_data = self.add_coords(coords)
+        self.edges_data = self.graph.get_edges()
+        
+        self.labels_color = labels_color
+        self.undiscovered_node_color=undiscovered_node_color 
+        self.discovered_node_outline=discovered_node_outline
+        self.current_node_fill=current_node_fill
+        self.discovered_node_fill=discovered_node_fill
+        self.undiscovered_node_fill=undiscovered_node_fill
+        self.completed_node_color=completed_node_color
+        self.undiscovered_edge_color=undiscovered_edge_color
+        self.discovered_edge_color=discovered_edge_color
+
+        self.add_node_colors()
+        self.add_edge_color()
+
+
+    def add_coords(self, coords):
+
+        node_data = []
+
+        if coords == None:
```

Comment:
In Python for that usually using `if coords is None`. Would work the same, but will be more readable